src/collections/_documentation/guides/getting-started/index.md

```diff
@@ -0,0 +1,118 @@
+
+You can also invite multiple new (external) members to join your organization and team by clicking on `Add Member > Invite Member`
+
+![Invite Members]({% asset guides/getting-started/05.png @path %})
+
+For more information see: [Organization Management](https://docs.sentry.io/accounts/membership/)
+
+## 3. Configure Integrations
+
+Sentry integrates seamlessly into your existing workflows by providing out of the box integrations with your favorite apps and services. To enable and configure integrations navigate to `Organization Settings > Integrations`.
+
+### 3.1. Alert Notifications
+
+By default, Sentry will notify you about errors in your apps via **email**. To receive real-time notifications about your errors where you work, enable Sentry integrations with: [Slack](https://docs.sentry.io/workflow/integrations/global-integrations/#slack), [PagerDuty](https://www.pagerduty.com/docs/guides/sentry-integration-guide/).
+
+> ![PagerDuty Integration]({% asset guides/getting-started/slack_sentry_e2e_integration4.gif @path %})
+>
+> _Enabling alert notifications through the __Slack__ integration._
+
+Sentry supports additional solutions on a project level. After creating a project - **Hipchat**, **Twilio**, **OpsGenie**, and other solutions can be configured and used to channel notifications through alert rules. For more information check out:
+
+- [Per-Project Integrations: Issue Notifications](https://docs.sentry.io/workflow/integrations/legacy-integrations/#issue-notifications)
+- [Routing Alerts to 3rd Party Integrations](https://docs.sentry.io/guides/alert-notifications/routing-alerts/#routing-alerts-to-3rd-party-integrations)
+
+### 3.2. Source Code Management
+
+By enabling Sentry's integration with your Source Code Management (SCM) provider, Sentry will:
+
+1. Identify **suspect commits** that likely introduced an error.
+2. **Suggest assignees** who can best resolve an error.
+3. Mark an issue as **Resolved via Commit or PR** to stop notifications about similar errors in past releases and notify you upon regression.
+4. Link a Sentry issue to a new or existing issue in your integrated [Issue Tracking](#23-issue-tracking) solution
+
+    ![Integrate SCMs]({% asset guides/getting-started/07.png @path %})
+
+Sentry has built-in integrations with: [GitHub](https://docs.sentry.io/workflow/integrations/global-integrations/#github), [GitHub Enterprise](https://docs.sentry.io/workflow/integrations/global-integrations/#github-enterprise), [GitLab](https://docs.sentry.io/workflow/integrations/global-integrations/#gitlab), and [Bitbucket](https://docs.sentry.io/workflow/integrations/global-integrations/#bitbucket). If you're using a different SCM provider or don't want Sentry to connect to your repository, checkout our [docs]((https://docs.sentry.io/workflow/releases/?platform=browser#alternatively-without-a-repository-integration)) to learn how you can still enable most of these features (#1 - #3) by telling Sentry about your commit metadata.
```

Comment:
```suggestion
Sentry has built-in integrations with: 
- [GitHub](https://docs.sentry.io/workflow/integrations/global-integrations/#github)
- [GitHub Enterprise](https://docs.sentry.io/workflow/integrations/global-integrations/#github-enterprise)
- [GitLab](https://docs.sentry.io/workflow/integrations/global-integrations/#gitlab) 
- [Bitbucket](https://docs.sentry.io/workflow/integrations/global-integrations/#bitbucket). 

If you're using a different SCM provider or don't want Sentry to connect to your repository, checkout our [docs]((https://docs.sentry.io/workflow/releases/?platform=browser#alternatively-without-a-repository-integration)) to learn how you can still enable most of these features (#1 - #3) by telling Sentry about your commit metadata.
```

Reply:
The GitHub and GitLab links can get confused with each other. Breaking them into bullet points might be easier to differentiate.